docs/modules/kubeedge.md

```diff
@@ -1,6 +1,6 @@
 # What is KubeEdge
 
-**KubeEdge** is an open source system extending native containerized application orchestration and device management to hosts at the Edge. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. It also supports MQTT and allows developers to author custom logic and enable resource constrained device communication at the Edge. Kubeedge consists of a cloud part and an edge part. The edge part has already been open sourced and the cloud part is coming soon!  
+**KubeEdge** is an open source system extending native containerized application orchestration and device management to hosts at the Edge. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. It also supports MQTT and allows developers to author custom logic and enable resource constrained device communication at the Edge. Kubeedge consists of a cloud part and an edge part. Both edge and cloud parts are now opensourced.

```

Comment:
Thanks for contribution @junxu 

Would you please update the Chinese README as well? https://github.com/kubeedge/kubeedge/blob/master/README_zh.md

Reply:
Do we need to mention this now : " Both edge and cloud parts are now opensourced."
I feel its ok to remove this now. earlier since cloud part was not opensourced , it was ok to mention.

Reply:
Thanks for you reply @m1093782566 !
The Chinese README is corresponding to https://github.com/kubeedge/kubeedge/blob/master/README.md, I think it's doesn't need to be updated, what's your opinion?

Reply:
Okay, fair enough.